pkg/manager/manager.go

```diff
@@ -60,19 +60,7 @@ func New(opts Options) (Manager, error) {
 	}
 
 	// Build the controller manager.
-	mgr, err := ctrl.NewManager(opts.KubeConfig, ctrl.Options{
-		Scheme:                  opts.Scheme,
-		MetricsBindAddress:      opts.MetricsAddr,
-		LeaderElection:          opts.LeaderElectionEnabled,
-		LeaderElectionID:        opts.LeaderElectionID,
-		LeaderElectionNamespace: opts.LeaderElectionNamespace,
-		SyncPeriod:              &opts.SyncPeriod,
-		Namespace:               opts.WatchNamespace,
```

Comment:
doesn't look like WatchNamespace is plumbed through to opts.Namespace anymore

Reply:
nice catch. :)